src/pages/Countries.js

```diff
@@ -0,0 +1,135 @@
+import { useState, useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import styled from 'styled-components';
+import { useParams } from 'react-router-dom';
+import { getCountries } from '../redux/countries';
+import { getContinents } from '../redux/continents';
+
+const Countries = () => {
+  const dispatch = useDispatch();
+  const { continent } = useParams();
+  const { continents } = useSelector((state) => state.Continents) || [];
+  const countryInfo = useSelector((state) => state.Countries);
+
+  const filteredContinent = continents.filter(
+    (item) => item.id === continent,
+  )[0];
+  const { countries } = filteredContinent || [];
+  const [country, setCountry] = useState();
+
+  useEffect(() => {
+    dispatch(getContinents());
+  }, []);
+
+  return (
+    <CountryWrapper>
+      <h1>{continent}</h1>
+      <Select name="country" onChange={(e) => setCountry(e.target.value)}>
+        <option value="" key="country">
+          Country...
+        </option>
+        {countries
+          && countries.map((land) => (
+            <option value={`${land}`} key={land}>
+              {land}
+            </option>
+          ))}
+      </Select>
+      <Button type="button" onClick={() => dispatch(getCountries(country))}>
+        <h4>Show Information</h4>
+      </Button>
+      {country && country === countryInfo.id ? (
+        <Details>
+          <li>
+            <h3>{countryInfo.id}</h3>
+          </li>
+          <li>
+            <img src={countryInfo.flag} alt="flag" width={150} />
+          </li>
+          <br />
+          <li>
+            <strong>Cases:</strong>
+            {countryInfo.cases}
+          </li>
+          <br />
+          <li>
+            <strong>Active:</strong>
+            {countryInfo.active}
+          </li>
+          <br />
+          <li>
+            <strong>Critical:</strong>
+            {countryInfo.critical}
+          </li>
+          <br />
+          <li>
+            <strong>Recovered:</strong>
+            {countryInfo.recovered}
+          </li>
+          <br />
+          <li>
+            <strong>Deaths:</strong>
+            {countryInfo.deaths}
+          </li>
+          <br />
+          <li>
+            <strong>Population:</strong>
+            {countryInfo.population}
+          </li>
+        </Details>
+      ) : (
+        'Choose a country'
+      )}
+    </CountryWrapper>
+  );
+};
```

Comment:
- [ ] Kindly follow the design of the template provided for the countries page. You can keep your dropdown and other feature, but please follow the design pattern used.

### Your Design

![image](https://user-images.githubusercontent.com/77106962/181652751-71e5a2fd-76ac-4df9-8dca-daa44f3d2165.png)

### Design Template

![image](https://user-images.githubusercontent.com/77106962/181652812-b9c3ae5c-21aa-4ce2-b6d9-b627f58439c1.png)